sdk/core/System.Net.ClientModel/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("System.Net.ClientModel.Tests.Internal, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4")]
```

Comment:
Internals are visible to the project specifically added for testing internal APIs.  This designation forces that separation, and we should try to keep tests of public API contracts out of this library if at all possible.  Dynamic types are the primary reason we want to enforce this, since we don't have compiler checking on those, and if we assume we're testing public APIs on dynamic types but tests pass because the test project has access to internals, we could inadvertently ship types to customers that they don't have access to use.